csi/node_server.go

```diff
@@ -442,6 +443,14 @@ func (ns *NodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 			return nil, status.Errorf(codes.InvalidArgument, "unsupported disk encryption format %v", diskFormat)
 		}
 
+		volumeSize, err := util.ConvertSize(volume.Size)
+		if err != nil {
+			return nil, status.Error(codes.Internal, err.Error())
+		}
+		if volumeSize <= crypto.Luks2MinimalVolumeSize {
+			return nil, status.Errorf(codes.Aborted, "unsupported volume size %v, need to be bigger than 16MiB default LUKS2 header size for encryption", volume.Size)
```

Comment:
Using `codes.InvalidArgument` instead.

Ref:
**InvalidArgument** indicates client specified an invalid argument.
**Aborted** indicates the operation was aborted, typically due to a concurrency issue like sequencer check failures, transaction aborts, etc

Reply:
Modified. Thanks for the explanation.